client/contracts/token/contracts/Token.Sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.4.21;
+pragma solidity ^0.4.21;
```

Comment:
This is usually a bad practise as we should use a concrete version

Reply:
Why do think that using ^ is bad practice? It helps greatly with specifying the minimal version our code needs to be compliant and if we want to update solidity, we do not need to change every contract.

Reply:
http://solidity.readthedocs.io/en/v0.4.24/layout-of-source-files.html#version-pragma

Reply:
https://github.com/ConsenSys/smart-contract-best-practices/blob/master/docs/recommendations.md
Section: Lock pragmas to specific compiler version

Reply:
Ok, got it. Will change that.